pkg/harvester/dialog/ConfirmRelatedToRemoveDialog.vue

```diff
@@ -145,6 +145,7 @@ export default {
       try {
         for (const resource of this.resources) {
           await resource.remove();
+          if (this.modalData.extraActionAfterRemove) await this.modalData.extraActionAfterRemove();
```

Comment:
[Nit] `extraActionAfterRemove` is optional, so use `this.modalData?.extraActionAfterRemove`.